### PR TITLE
Add token documentation

### PR DESCRIPTION
**Pull request by stefannibrasil**
This PR adds the token documentation discussed in #2018 about people or bots commenting via API using the token system

### CODE REVIEW

**Comment by stefannibrasil** [comment]
@ryzokuken @publiclab/reviewers could you please review here? 

@sukhbir-singh and I would love to get some feedback about it.

File: doc/API.md [hunk 1]
```diff
@@ -57,3 +57,44 @@ And several tag-based JSON/XML listings are generated directly from controllers,
...
+* **Success Response:**
+  * **Code:** 201 Created <br>
+    **Content:** None
+
+* **Error Response:**
+  * **Code:** 400 BAD REQUEST <br>
+    **Content:** None
+
+* **Sample Call:**
+  ```POST https://publiclab.org/comment/create/token/id.:format
```
Comment (ryzokuken) [inline doc/API.md]: This needs to go on different lines, with a newline between the backticks and the text.

**Comment by sukhbir-singh** [comment]
@ryzokuken I have appended changes requested by you to last commit in this pull request.

**Comment by jywarren** [comment]
This is fantastic!! Thank you!!

**Pull request merged**